apps/prairielearn/src/schemas/schemas/infoCourse.json

```diff
@@ -160,6 +160,25 @@
           }
         }
       }
+    },
+    "sharingSets": {
+      "description": "Sharing sets",
+      "type": "array",
+      "items": {
+        "description": "A sharing set description.",
+        "type": "object",
+        "required": ["name"],
+        "properties": {
+          "name": {
+            "description": "Name of the sharing set.",
+            "type": "string"
+          },
+          "description": {
+            "description": "Description of the sharing set.",
+            "type": "string"
+          }
```

Comment:
Checking my understanding: this would be unused in the current DB schema, correct? Are there plans to support this soon?

Reply:
that's correct, its not used in the DB right now. I just put it there because Matt and I figured we would want it at some point, but we can change things if we want.